### DISCUSSION TITLE
Cruiser uninstall > keep data ?

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#80] [Q&A] [answered]
https://www.androidpolice.com/2019/11/06/some-apps-like-whatsapp-offer-to-save-app-data-when-uninstalled-in-android-10/

Does this also still apply on A11 and next ? 
To Keep at least the LARGE vector maps or other important data.

**Keep Android > data > gr.talent.cruiser**
and or 
**Keep Android > media  > gr.talent.cruiser**

**Comment by devemux86** [comment]
@0709wiwiwi thanks for the suggestion.

We could do this. It is available in Android 10 and higher.

It will appear that way to users:

<img src="https://user-images.githubusercontent.com/3484020/224542808-9ac21023-d464-443a-b493-a8496b477a2c.jpg" width="50%"/>

**Comment by devemux86** [comment, marked as answer]
Available in [Cruiser 3.0.13](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by 0709wiwiwi** [comment]
**So that article description in the link looked very promising.** 
Not as expected causes even more misery. Do not use no succes.   :-((

**Reply by devemux86** [reply]
@0709wiwiwi 

> data in Android > media

Have you tested with the default `Android/data` ?

**Reply by 0709wiwiwi** [reply]
By Android/data also was not successful.   :-((
You get stuck later if you accept the question save data ?
Refuse !!!   **Do NOT say yes** !

**Reply by devemux86** [reply]
Looks like an Android problem, apps just enable the flag.
Everything is managed by Android, apps are not involved.

**Reply by devemux86** [reply]
In any case and on all operating systems the recommended way is always a clean install.
Leftovers from previous installations can only cause problems.

Also most users do not need to reinstall Cruiser anyway. :slightly_smiling_face:
